waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -52,9 +53,14 @@ migrateDevAndCopyToSource dbMigrationsDirInWaspProjectDirAbs genProjectRootDirAb
     ExitFailure code -> return $ Left $ "Migrate (dev) failed with exit code: " ++ show code
 
 finalizeMigration :: Path' Abs (Dir ProjectRootDir) -> Path' Abs (Dir DbMigrationsDir) -> IO (Either String ())
-finalizeMigration genProjectRootDirAbs dbMigrationsDirInWaspProjectDirAbs =
-  copyMigrationsBackToSource genProjectRootDirAbs dbMigrationsDirInWaspProjectDirAbs
-    <* writeDbSchemaChecksumToFile genProjectRootDirAbs (SP.castFile dbSchemaChecksumOnLastMigrateFileProjectRootDir)
+finalizeMigration genProjectRootDirAbs dbMigrationsDirInWaspProjectDirAbs = do
```

Comment:
huh this one was a bit hard to follow for me :D Here's what I understood:
- first it removes migrations dir from checksum file, so we'll have to write to the disk again during the next compilation -> why is that important? I saw the note, but couldn't manage to construct the "bad scenario" that we want to avoid in my head
- then it copies the migrations folder from out/generated dir to the wasp project dir
- it writes db schema checksum to some separate file? So we have another checksum file just for db schema, or is that the part of the original .waspchecksums file?